roles/openshift_expand_partition/tasks/main.yml

```diff
@@ -1,6 +1,14 @@
 ---
 - name: Ensure growpart is installed
   yum: pkg=cloud-utils-growpart state=present
+  when: not is_containerized | bool
+
+- name: Determine if growpart is installed
+  command: "rpm -q cloud-utils-growpart"
+  register: has_growpart
+  failed_when: "has_growpart.cr != 0 and 'package cloud-utils-growpart is not installed' not in has_growpart.stdout"
+  changed_when: false
+  when: is_containerized | bool
```

Comment:
Are we actually making use of this? I'd rather lean on cloud-init to handle growing the partition rather than us attempting to do it for the user.

I can see Ops wanting to automate this for their systems, but it seems like something we wouldn't want to support from the OpenShift installation/configuration side.